learn/configuration/typo_tolerance.md

```diff
@@ -0,0 +1,87 @@
+
+Meilisearch won't apply typo tolerance on the query term `Shrek` or `shrek` at search time to match documents.
+
+::: note
+`disableOnWords` is case insensitive.
+:::
+
+### `disableOnAttributes`
+
+You can disable the typo tolerance feature for a specific document attribute by adding it to `disableOnAttributes`. The code sample below disables typo tolerance for `title`:
+
+<CodeSamples id="typo_tolerance_guide_2" />
+
+If you type `Dunbo` instead of `Dumbo` with the above settings, Meilisearch won't match any documents.
+
+## Impact of typo tolerance on the `typo` ranking rule
+
+The [`typo` ranking rule](/learn/core_concepts/relevancy.md#_2-typo) sorts the results by increasing number of typos on matched query words. Documents with 0 typos will rank highest. This rule does not impact the typo tolerance setting.
+
+If you don't use the `typo` ranking rule but enable typo tolerance for an index, Meilisearch will use typo tolerance to match documents but won't sort them based on increasing number of typos.
+
+## How are typos calculated
+
+Typo tolerance is applied before sorting documents. It aggregates them and chooses which documents contain words similar to the queried words. Meilisearch then uses a prefix [Levenshtein algorithm](https://en.wikipedia.org/wiki/Levenshtein_distance) to check if the words match. It accepts every word that **starts with the query words or has the same length**.
+
+The Levenshtein distance between two words _M_ and _P_ is called "the minimum cost of transforming _M_ into _P_" by performing the following elementary operations:
+
+- substitution of a character of _M_ by a character other than _P_ (e.g., **k**itten → **s**itten)
+- insertion in _M_ of a character of _P_ (e.g., siting → sit**t**ing)
+- deletion of a character from _M_ (e.g., satu**r**day → satuday)
+
+By default, Meilisearch uses the following rules for matching documents, you can configure them using the [update typo tolerance endpoint](/reference/api/typo_tolerance.md#update-typo-tolerance). These rules are **by word** and not for the whole query string.
+
+- If the query word is between `1` and `4` characters, **no typo** is allowed. Only documents that contain words that **start with** or are of the **same length** with this query word are considered valid
+- If the query word is between `5` and `8` characters, **one typo** is allowed. Documents that contain words that match with **one typo** are retained for the next steps.
+- If the query word contains more than `8` characters, we accept a maximum of **two typos**
+
+This means that "saturday" which is `7` characters long, uses the second rule and matches every document containing **one typo**. For example:
+
+- "saturday" is accepted because it is the same word
+- "satuday" is accepted because it contains **one typo**
+- "s**u**tuday" is not accepted because it contains **two typos**
+- "**c**aturday" is not accepted because it contains **two typos**
```

Comment:
I'm not opposed to having `caturday` as an example, but we it doesn't make sense to have both `satuday` and `caturday`.

Also, the description is wrong (caturday contains only one typo).

```suggestion
```

Reply:
Disagree, `caturday` is an effective example because a typo on the first letter of a word counts as two typos.